pkg/pillar/docs/zedmanager.md

```diff
@@ -0,0 +1,46 @@
+# Application Instance Management Agent in EVE (aka zedmanager)
+
+## Overview
+
+Zedmanager does the overall orchestration of application instances in EVE.
+This means receiving configuration from the controller via zedagent, and reporting results to the controller via zedagent, and then using other EVE microservices to manage the different parts of the orchestration.
+
+- Using volumemgr to create the (disk) volumes for the application instance
+- Using identitymgr to create any overlay EIDs needed for overlay networking
```

Comment:
Can you expand "EIDs"? Many may know what it is, but since this is the entry point for apps into a device, I wouldn't be surprised to find people starting here for the first time.

Reply:
Expanded and added reference